resources/rules.k.tmpl

```diff
@@ -213,19 +221,21 @@ rule ACCTCODE in SetItem( 1 )
     rule 0 <=Int keccakIntList(N)     => true
     rule keccaktIntList(N) <Int pow256 => true
 
-
-    rule #padToWidth(32, #asByteStack( #unsigned(V) )) => #asByteStackInWidth( #unsigned(V), 32)
-      requires #rangeSInt(256, V)
-
-
     rule #padToWidth(32, #asByteStack(V)) => #asByteStackInWidth(V, 32)
       requires 0 <=Int V andBool V <Int pow256
 
+    rule #padToWidth(32, #asByteStack(V)) => #asByteStackInWidth(V, 32)
+      requires 0 <=Int V andBool V <Int pow160
```

Comment:
Does this make sense when the rule above includes this one?

Reply:
@desaperados ?
I'll remove it and see if something crashes..

Reply:
@desaperados ?
I'll remove it and see if something crashes..